build.py

```diff
@@ -1,7 +1,7 @@
 import subprocess
 import os
 import sys
-from shutil import copytree
+from distutils.dir_util import copy_tree
```

Comment:
this was merged into upstream master. 
pull from master to get these changes, instead of re-committing them..